cimetrics/upload.py

```diff
@@ -52,11 +56,12 @@ def publish(self):
         }
         if self.env.is_pr:
             doc["target_branch"] = self.env.target_branch
+
         coll.insert_one(doc)
 
 
 @contextlib.contextmanager
-def metrics():
-    m = Metrics()
+def metrics(complete: bool = True) -> None:
+    m = Metrics(complete=complete)
```

Comment:
We could forward everything, but this is more user/autocomplete friendly, and we don't have enough arguments that it's a problem.